nixos/modules/services/networking/hylafax/systemd.nix

```diff
@@ -7,12 +7,30 @@
 
 let
 
+  inherit (lib.cli) toGNUCommandLine;
+  inherit (lib) optional;
   inherit (lib) mkIf mkMerge;
-  inherit (lib) concatStringsSep optionalString;
+  inherit (lib)
+    concatLines
+    concatStringsSep
+    escapeShellArgs
+    optionalString
+    ;
```

Comment:
This can all be taken directly from lib

Reply:
`toGNUCommandLine` must be taken from `lib.cli`, it's also missing in the `lib` namespace.